ert3/evaluator/_evaluator.py

```diff
@@ -34,6 +34,11 @@ def _prepare_input(ee_config, stages_config, inputs, evaluation_tmp_dir):
     tmp_input_folder = evaluation_tmp_dir / "prep_input_files"
     os.makedirs(tmp_input_folder)
     ee_storage = storage_driver_factory(ee_config["storage"], tmp_input_folder)
+    # record2location = {}
+    # for idx in range(0, len(stages_config)-1):
+    #    record2location.update(
+    #        {input.record: input.location for input in stages_config[idx].input}
+    #    )
     record2location = {input.record: input.location for input in stages_config[0].input}
```

Comment:
Not sure why we always use the input of stage 0  (stages_config[0].input). What if I want to have 2 stages with different input, does that not make sense ?

Reply:
I don't know why, but a comment a few lines up says that only one stage is supported at the moment.
What is you addition about, and why is it commented out?